wp-sw-cache/wp-sw-cache-main.php

```diff
@@ -9,11 +10,9 @@ class SW_Cache_Main {
   private static $instance;
 
   public function __construct() {
-    if($this->show_header()) {
-      add_action('wp_head', array($this, 'register')); // TODO:  Find out why "wp_footer" isn't working
+    if (get_option('wp_sw_cache_enabled')) {
```

Comment:
I've updated the PR to re-enable your option for enabling / disabling the service worker.